kubemarine/procedures/check_paas.py

```diff
@@ -217,37 +218,49 @@ def check_packages_versions(cluster: KubernetesCluster, tc: TestCase, hosts_to_p
     :param cluster: main cluster object
     :param tc: current test case object
     :param hosts_to_packages: hosts where to check packages
-    :param warn_on_bad_result: if true then uses Warning instead of Failure. Default False.
+    :param warn_on_bad_result: if true then bad results raises Warning instead of Failure. Default False.
+    :param raise_successful: if true, successful test result will be risen. Default True.
     """
     bad_results = []
+    warn_results = []
     good_results = []
 
     packages_map = pckgs.detect_installed_packages_version_hosts(cluster, hosts_to_packages)
+    packages_section = cluster.inventory['services']['packages']
+    global_cache_versions = packages_section['cache_versions']
     for package, version_map in packages_map.items():
+        package_associations = packages_section['associations'].get(cluster.get_os_family(), {}).get(
+            get_package_name(cluster.get_os_family(), package), {})
```

Comment:
`get_package_name(cluster.get_os_family(), package)` is a package name, not an association key. For example, on rhel-like OS families it will be `containerd.io` for containerd.

Reply:
Fixed in https://github.com/Netcracker/KubeMarine/pull/649/commits/ab8daf58f44c420932b99c7c785ef1926d393829